sources/models/countries.js

```diff
@@ -1,5 +1,13 @@
-export const countries = [
-	{"id":1,"Name":"USA"},
-	{"id":2,"Name":"Canada"},
-	{"id":3,"Name":"Italy"}
-];
\ No newline at end of file
+// export const countries = [
+// 	{"id":1,"Name":"USA"},
+// 	{"id":2,"Name":"Canada"},
+// 	{"id":3,"Name":"Italy"}
+// ];
```

Comment:
remove commented lines

Reply:
I know it, sorry. Tested a lot of different features and in the end forgot to remove them.